lib/snippet.js

```diff
@@ -0,0 +1,72 @@
+import { IdentityVerification } from './index';
+import htmlencode from 'htmlencode';
+
+export default class Snippet {
+  constructor(settings) {
+    if (!settings.app_id) {
+      throw new Error('You must provide an app_id in your Intercom settings');
+    }
+    if (!settings.verificationSecret) {
+      throw new Error('You must provide your verification secret in your Intercom settings');
+    }
+    if (!settings.user_id && !settings.email) {
```

Comment:
Can we make this snippet work for logged out mode as well?

Reply:
Yep! I'll add that.

Reply:
This supports logged out now.